README.md

```
@@ -1,302 +1,191 @@
-# <img src="./docs/logo.png" alt="Tsarr Logo" width="40" height="40" style="vertical-align: middle; margin-right: 8px;"> Tsarr
+<div align="center">
+  <img src="./docs/logo.png" alt="Tsarr" width="96" height="96" />
 
-*TypeScript-arr (pronounced "Tsar" /tsɑr/ - a Slavic king/emperor)*
+  # Tsarr
```

markdownlint-cli2 (0.22.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)